src/common/transformations/src/transformations/common_optimizations/remove_concat_zero_dim_input.cpp

```diff
@@ -45,7 +45,19 @@ ov::pass::RemoveConcatZeroDimInput::RemoveConcatZeroDimInput() {
 
         bool inputs_removed = concat->get_input_size() > concat_inputs.size();
         if (inputs_removed) {
-            concat->set_arguments(concat_inputs);
+            if (concat_inputs.empty()) {
```

Comment:
but the problem with empty input tensors to concat op still exists, right?
I mean the situation when not all tensors to concat are empty but some of them and they are cut. In some cases these empty tensors affect concat output shape. 
do we plan to fix this?

Reply:
I don't plan to fix this in current release as we know there could be drawbacks. As @slyalin proposed -- let's fix this model in current release and plan axis fix for future releases

Reply:
ok, could you create a jira ticket?

Reply:
Done